DeskReserve/Domain/BuildingService.cs

```diff
@@ -0,0 +1,70 @@
+﻿using DeskReserve.Data.DBContext.Entity;
+using DeskReserve.Data.DBContext;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.AspNetCore.Mvc;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace DeskReserve.Domain
+{
+    public class BuildingService
+    {
+        private readonly ApplicationDbContext _dbContext;
+
+        public BuildingService(ApplicationDbContext dbContext)
+        {
+            _dbContext = dbContext ?? throw new ArgumentNullException(nameof(dbContext));
+        }
+
+        public List<Building> GetAllBuildings()
```

Comment:
Return a task and make this asynchronous. We need to enable simultaneous handling of multiple requests.